### PR TITLE
Added 'employeeLeaveDateTime' to 'User' object.

### PR DESCRIPTION
**Pull request by rajadineshmurugesan-microsoft**

### CODE REVIEW

File: concepts/tutorial-lifecycle-workflows-set-employeeleavedatetime.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ PATCH https://graph.microsoft.com/beta/users/df744d9e-2148-4922-88a8-633896c1e92
 # [PowerShell](#tab/powershell)
 
 ```powershell    
-    Connect-MgGraph -Scopes "User-LifeCycleInfo.ReadWrite.All"
+    Connect-MgGraph -Scopes "User.Read.All","User-LifeCycleInfo.ReadWrite.All"
     Select-MgProfile -Name "beta"
```
Comment (FaithOmbongi) [inline concepts/tutorial-lifecycle-workflows-set-employeeleavedatetime.md:47]: Update to use v1.0
```suggestion
    Select-MgProfile -Name "v1.0"
```

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
Thank you @rajadineshmurugesan-microsoft for your work on this PR.

I've suggested some edits to the descriptions in beta. Please apply the edits to v1.0. Also left some other minor change requests.

**Comment by OWinfreyATL** [comment]
Hey @FaithOmbongi 

Is there a way to see where an error is in the changelog? Started getting this vague error:
image

When merging suggested changes into this PR.

thanks.

**Comment by FaithOmbongi** [comment]
> Hey @FaithOmbongi
> 
> Is there a way to see where an error is in the changelog? Started getting this vague error: image
> 
> When merging suggested changes into this PR.
> 
> thanks.

I think it's because of the merge conflict in the changelog. Try resolving that and see if it fixes the issue.

**Comment by FaithOmbongi** [comment]
Hi @OWinfreyATL - Please apply the changes in beta to the v1.0 docs as well.

**Review by FaithOmbongi — APPROVED** [review]
LGTM! Approved!

**Pull request merged**